static/js/containers/PostPage.js

```diff
@@ -158,6 +187,15 @@ class PostPage extends React.Component<*, void> {
       <div>
         <ChannelBreadcrumbs channel={channel} />
         <DocumentTitle title={formatTitle(post.title)} />
+        <Dialog
+          open={showDialog}
+          onCancel={() => this.setShowDeletePostDialog(false)}
```

Comment:
You can change this to `hideDialog={...}` and then you don't need to also hide the dialog in `deletePost`

Reply:
oh nice! I will do that.